src/ReactiveDomain.Users/Domain/Client.cs

```diff
@@ -0,0 +1,119 @@
+﻿using ReactiveDomain.Messaging;
+using ReactiveDomain.Util;
+using System;
+using System.Collections.Generic;
+using static IdentityModel.OidcConstants;
+using static ReactiveDomain.Users.Messages.ClientMsgs;
+
+namespace ReactiveDomain.Users.Domain
+{
+
+    public class Client : AggregateRoot
+    {
+        public string ClientName { get; private set; }
+        public string[] RedirectUris { get; private set; }
+        public string[] LogoutRedirectUris { get; private set; }
+        public string FrontChannlLogoutUri { get; private set; }
+
+        private Client()
+        {
+            RegisterEvents();
+        }
+
+        private void RegisterEvents()
+        {
+            Register<ClientCreated>(@event =>
+            {
+                Id = @event.ClientId;
+                ClientName = @event.ClientName;
+                RedirectUris = @event.RedirectUris;
+                LogoutRedirectUris = @event.PostLogoutRedirectUris;
+                FrontChannlLogoutUri = @event.FrontChannelLogoutUri;
+            });
+            Register<ClientSecretAdded>(@event => { });
+            Register<ClientSecretRemoved>(@event => { });
+        }
+
+        ////constructor for Elbe client
+        //public Client(
```

Comment:
Can this commented stuff all be deleted?

Reply:
It captures certain settings I'd rather not remove until we've got things finalized.